ext/sst/Makefile

```diff
@@ -1,9 +1,9 @@
 # These two variables are designed to be modifiable.
-SST_VERSION=SST-trunk
+SST_VERSION=${HOME}/fast/sst/lib/pkgconfig/SST-10.0.0.pc
```

Comment:
Probably don't want to hardcode this for your setup, generally

Reply:
yeah, I haven't found a better way so far